lib/train/transports/ssh.rb

```diff
@@ -43,11 +43,11 @@ class SSH < Train.plugin(1) # rubocop:disable Metrics/ClassLength
 
     # common target configuration
     option :host,      required: true
-    option :port,      default: 22, coerce: proc { |u| read_options_from_ssh_config(u, :port) }, required: true
-    option :user,      default: "root", coerce: proc { |u| read_options_from_ssh_config(u, :user) }, required: true
+    option :port,      default: 22, required: true
+    option :user,      default: "root", required: true
     option :key_files, default: nil
     option :password,  default: nil
-
+    option :ssh_config, default: false
```

Comment:
I suggest renaming this option to be :ssh_config_file and adding a comment explaining that you may pass the value `true` to use the default location.